src/content/docs/data-apis/manage-data/manage-data-retention.mdx

```diff
@@ -508,6 +525,9 @@ This table shows the default [namespace](/docs/glossary/glossary) retention sett
   </tbody>
 </table>
 
+* This applies to metadata, stack traces can only be retained for up to 8 days. 
```

Comment:
what is this referring to? didn't see an asterisk up above? I'd be happy to review this one as the data retention space is a bit of a messy area that I've worked on a lot.

Reply:
oh nevermind I saw austin's note about not wanting the footnote